src/odemis/acq/test/move_test.py

```diff
@@ -541,7 +543,7 @@ def test_sample_switch_procedures(self):
         self.assertEqual(pos_label, FM_IMAGING)
 
         # Test the progress update
-        progress_fm = getMovementProgress(stage.position.value, self.stage_coating, self.stage_active)
+        progress_fm = getMovementProgress(stage.position.value, self.stage_deactive, self.stage_active)
```

Comment:
why use stage_deactive here, while on line 528 stage_coating is replaced by stage_active

Reply:
It's because now that coating is the same as active, we need to test something else. Like the move between deactive and active. However, leaving this check here is confusing. I'll move it.